app/partials/register-form.cjsx

```diff
@@ -28,6 +28,7 @@ counterpart.registerTranslations 'en',
     email: 'Email address'
     emailConflict: 'An account with this address already exists'
     realName: 'Real name'
+    realNamePatternHelp: "Enter a real name, not an email address", 
```

Comment:
```suggestion
    realNamePatternHelp: "Enter a name, not an email address", 
```

Reply:
I copied over what I used in the new app, so I'll have to make this change there too.

Reply:
It's not a big deal. It occurred to me because I sometimes get told by web forms that O'Donnell is 'not a real name.'

Reply:
It's probably best to avoid this language since folks might want to use a pseudonym or a nickname or something. 👍